### PR TITLE
Update OpenBSDEurope.com.xml and OpenBSD_Store.com.xml

### PR DESCRIPTION
**Pull request by koops76**
#11131

### CODE REVIEW

File: src/chrome/content/rules/OpenBSDEurope.com.xml [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
 <ruleset name="OpenBSD Europe">
-  <target host="openbsdeurope.com" />
-  <target host="shop.openbsdeurope.com" />
+	<target host="openbsdeurope.com" />
+	<target host="shop.openbsdeurope.com" />
 
-  <rule from="^http://(?:shop\.)?openbsdeurope\.com/" to="https://shop.openbsdeurope.com/"/>
+	<securecookie host="^shop\.openbsdeurope\.com$" name=".+" />
+	
+	<rule from="^http://openbsdeurope\.com/" to="https://shop.openbsdeurope.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/OpenBSDEurope.com.xml]: Please add a hint about the mismatch

Comment (J0WI) [inline src/chrome/content/rules/OpenBSDEurope.com.xml]: See also `www.openbsdeurope.com`

Comment (koops76) [inline src/chrome/content/rules/OpenBSDEurope.com.xml]: @J0WI Done.

**Comment by J0WI** [comment]
Please have a look at Travis

**Comment by koops76** [comment]
@J0WI Request review.

**Comment by J0WI** [comment]
Tests are still broken.

**Pull request closed without merging**